l10n_it_fatturapa_out_report/models/report.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+from openerp import models, api, _
+import base64
+import openerp
+
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    @api.v8
+    def generate_attach_report(self, inv):
```

Comment:
@sergiocorato riesci a chiamare `super` in caso di report standard e chiamare questa nuova parte solo nel caso in cui serve? (evidenziando con dei commenti cosa cambia rispetto al metodo originale)

Reply:
@eLBati cosa dici della #780 ? sostituisce questa correggendo direttamente il modulo l10n_it_fatturapa_out, in alternativa c'è la #770 , aspettavo il test di @d-stragapede se risolve anche a lui